### DISCUSSION TITLE
Where to suggest doc edit? (replace 'accounts' with 'trades' in this case)

### DISCUSSION THREAD

**Discussion by lanceculnane** [hootnot/oanda-api-v20#186] [General]
Sorry- idk if you deal with the documentation on oanda-api-v20 (or where I can pass this on) but I think 'accounts' in the 4th line should be replaced with 'trades'

>>> import oandapyV20
>>> import oandapyV20.endpoints.trades as trades
>>> client = oandapyV20.API(access_token=...)
>>> r = accounts.TradeDetails(accountID=..., tradeID=...)
>>> client.request(r)
>>> print r.response

https://oanda-api-v20.readthedocs.io/en/latest/endpoints/trades/tradedetails.html
Thanks! I appreciate all your work- super amazing and helpful!

**Comment by hootnot** [comment]
You are right. I'll fix it. Thanks

(fixed + new version 0.7.2)
